docs/src/components/ExternalExampleLayout.tsx

```diff
@@ -2,19 +2,23 @@ import * as _ from 'lodash/fp'
 import PropTypes from 'prop-types'
 import * as React from 'react'
 
-import { exampleContext } from 'docs/src/utils'
+import { exampleContext, exampleKebabNameToFilename, parseExamplePath } from 'docs/src/utils'
 import PageNotFound from '../views/PageNotFound'
 
-const exampleKeys = exampleContext.keys()
+const examplePaths = exampleContext.keys()
 
 const ExternalExampleLayout: any = props => {
   const { exampleName } = props.match.params
-  const exampleFilename = `/${_.startCase(exampleName).replace(/ /g, '')}.tsx`
+  const exampleFilename = exampleKebabNameToFilename(exampleName)
 
-  const componentKey = _.find(_.endsWith(exampleFilename), exampleKeys)
-  if (!componentKey) return <PageNotFound />
+  const examplePath = _.find(path => {
+    const { exampleName } = parseExamplePath(path)
+    return exampleFilename === exampleName
+  }, examplePaths)
```

Comment:
Fixed the maximize view.  It didn't handle `.shorthand.tsx` files.  Now it does.